docs/development/maintenance.md

```diff
@@ -105,6 +105,39 @@ addition the company the person is working for can be listed, too.
 Authorities** the company can propose a prioritized candidate for the
 succession.
 
+## Process to Change Authorities
+* To apply for an `Authority` of a `Component` that currently has a holder of the
+`Authority to (De-)Assign Authorities`, it is recommended to contact this person
+before taking the next step.
+* Please provide a Pull Request against the `trunk`-branch of the [official ILIAS Repository](https://github.com/ILIAS-eLearning/ILIAS)
+to change assignments to `Authorities` for some `Component`. Please explain in
+its message why this change hould be made. Also shortly report your exchange with
```

Comment:
Just to avoid a possible misunderstanding on my end: this refers to the message of the _pull request_, not the _commit message_, correct? (No objections to either at the moment; I may need to think about it once more.)